validator/src/validation/cards.rs

```diff
@@ -0,0 +1,29 @@
+use validation::Validator;
+use card_validate::Validate;
+
+
+pub fn validate_credit_card(card: &str) -> bool {
+    println!("{:?}", card);
+    match Validate::from(card) {
```

Comment:
I think `Validate::from(card).map(|v| v.valid).unwrap_or(false))` would be cleaner